internal/state/indexer/block/kv/kv.go

```diff
@@ -364,7 +364,6 @@ func (idx *BlockerIndexer) Search(ctx context.Context, q *query.Query) ([]int64,
 
 		select {
 		case <-ctx.Done():
-			break
```

Comment:
we need to revisit these breaks as it might be a developer error ⚠️

Reply:
on my part, or on the original author?

I had same "revisit" vibe.

Reply:
that is to say that I am not 100% sure I made a mistake with them.  

but could have.

Reply:
>  on the original author?

original author